.rubocop.yml

```diff
@@ -4,6 +4,9 @@ AllCops:
     - '**/Rakefile'
     - 'vendor/**/*'
 
+Style/Lambda:
+  Enabled: false
```

Comment:
I think this allows for shorter line lengths and the stabby syntax is easy to comprehend. Thoughts.

Reply:
This works for me 👌